test/vm.install

```diff
@@ -4,9 +4,10 @@
 set -eu
 
 # don't force https:// (self-signed cert)
+mkdir -p /etc/cockpit
 printf "[WebService]\\nAllowUnencrypted=true\\n" > /etc/cockpit/cockpit.conf
 
-if type firewall-cmd >/dev/null 2>&1; then
+if systemctl is-active -q firewalld.service; then
```

Comment:
Thanks -- this makes so much more sense, we should do that in the other projects, too.